test/Connector.t.sol

```diff
@@ -50,8 +50,8 @@ contract ConnectorTest is Test {
         assertTrue(token_ != address(0));
 
         RestrictedTokenLike token = RestrictedTokenLike(token_);
-        assertEq(token.name(), tokenName);
-        assertEq(token.symbol(), tokenSymbol);
+        assertEq(token.name(), bytes32ToString(stringToBytes32(tokenName)));
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
Sure! Actually I spent a lot of time thinking about this one... I ended up on the fact that probably the current behaviour is correct (where a string of "\0\0\0\0" gets treated as an empty string), so the problem then was that the test didn't reflect that. Do you agree?

I'll add a comment explaining whats going on here though